app/commands/mentor/update_stats.rb

```diff
@@ -1,7 +1,11 @@
-class UpdateMentorStatsJob < ApplicationJob
+class Mentor::UpdateStats
```

Comment:
As this wasn't used in our sidekiq schedule, I snuck in this additional refactoring to make this a mandate job

Reply:
There's no syntax highlighting in this file for some reason. Double-check the file extension doesn't have any hidden unicode pls.